CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+# Code of Conduct
+
+Kubeflow Spark Operator follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
```

Comment:
Please can we refer the [Kubeflow website code of conduct](https://www.kubeflow.org/docs/about/contributing/#follow-the-code-of-conduct).
So we have 1 place where we should redirect contributors.